gbus/worker.go

```diff
@@ -166,6 +172,27 @@ func (worker *worker) resolveHandlers(isRPCreply bool, delivery amqp.Delivery) [
 	return handlers
 }
 
+func exchangeAndRoutingFromDelivery(delivery amqp.Delivery) (exchange string, routingKey string, err error) {
```

Comment:
Please add commentary as to why we are performing this logic, as this is a bit "tricky" and highly contextual please over-communicate here.
In addition, I will create a bug report and we will add the link to it here